test/test_jit_cuda_fuser.py

```diff
@@ -761,6 +762,26 @@ def t(x: torch.Tensor, y: torch.Tensor, z: torch.Tensor):
         self.assertEqual(o, jit_o)
         self.assertGraphContains(t_jit.graph_for(x, y, z), FUSION_GUARD)
 
+    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
+    @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.PROFILING,
+                     "Requires fusion optimization pass to be effective")
+    def test_sum_to_one(self):
+        dtype = torch.float
+        device = "cuda"
+        x = torch.randn([4, 5, 6], dtype=dtype, device=device)
+
+        def t(x: torch.Tensor):
+            o = torch.add(x, 0)
+            o = torch.sum(o, dim=[0, 1, 2])
```

Comment:
Note to myself that we should update parser to enable sum to scalar for the overload of `torch.sum`.